identity/app/actions/AuthenticatedActions.scala

```diff
@@ -101,7 +101,7 @@ class AuthenticatedActions(
             Future.successful(Right(new AuthenticatedRequest(userFormCookie, request)))
 
           case _ =>
-            checkRecentAuthenticationAndRedirect(request)
+            Future.successful(Left(sendUserToSignin(request)))
```

Comment:
I think this means that people will be able to navigate to email prefs without the recentlyAuthenticated check because `consentAuthRefiner` is used by `consentJourneyRedirectAction`.